contracts/EverlongStrategy.sol

```diff
@@ -502,7 +508,7 @@ contract EverlongStrategy is BaseStrategy {
             // Hyperdrive's minimum transaction amount.
             if (
                 totalPositionValue >
-                (_targetOutput - output + _minimumTransactionAmount()).mulUp(
+                (_targetOutput - output + minimumTransactionAmount()).mulUp(
```

Comment:
Edit: I'll comment this on #49 instead and should be fixed there imo.

when you're closing longs, the minimum tx amount refers to **bond** amounts, you don't need to do any conversions then. just use `_poolConfig.minimumTransactionAmount`.

See https://github.com/delvtech/hyperdrive/blob/02d3cd465afbf86cc5f490c618a3a850c6dc6bfe/contracts/src/internal/HyperdriveLong.sol#L167-L171:

```solidity
if (_bondAmount < _minimumTransactionAmount) {
    revert IHyperdrive.MinimumTransactionAmount();
}
```